Marlin/ultralcd.cpp

```diff
@@ -1969,13 +1969,17 @@ void kill_screen(const char* lcd_msg) {
         print_job_counter.loadStats();
         printStatistics stats = print_job_counter.getStats();
 
-        char timeString[8];
-        sprintf_P(timeString, PSTR("%i:%02i"), int(stats.printTime / 60 / 60), int(stats.printTime / 60) % 60);
+        int min = long(stats.printTime / 60) % 60;
+        int hrs = long(stats.printTime / 60 / 60) % 24;
+        int day = int(stats.printTime / 60 / 60 / 24);
+        char timeString[14];
+        sprintf_P(timeString, PSTR("%i%s %i%s %02i%s"), day, MSG_SHORT_DAY, hrs, MSG_SHORT_HOUR, min, MSG_SHORT_MINUTE);
```

Comment:
@thinkyhead OK, you got me. Tried that previously, but I thought, I can't do modulus (%) with decimals or whatewer it is. And converting to long caused weird bugs inside Arduino, so I used temporary vars. Now OK?